sites/public/src/components/directory/ListingCard.module.scss

```diff
@@ -0,0 +1,200 @@
+.list-item {
+  &:not(:last-child) {
+    margin-block-end: var(--seeds-spacer-content);
+  }
+
+  // Don't put the focus state on the listing name link, because we are redirecting visual focus to the full card
+  :focus-within a:focus {
+    outline: none;
+    box-shadow: none;
+  }
+
+  // When the listing link is focused, put the focus style instead on the whole card
+  :focus-within.listing-card-container {
+    outline: var(--seeds-focus-ring-outline);
+    box-shadow: var(--seeds-focus-ring-box-shadow);
+    box-sizing: border-box;
+  }
+
+  // If a button inside the card is focused, turn the card-level focus, so only the button has visual focus
+  :has(button:focus).listing-card-container {
+    outline: none;
+    box-shadow: none;
+  }
+
+  .listing-card-container {
+    --card-spacing: 0;
+    position: relative;
+
+    .name,
+    a {
+      color: var(--seeds-color-blue-500);
+    }
+
+    &:hover {
+      outline: var(--seeds-border-2) solid var(--seeds-border-color);
+      box-sizing: border-box;
+      .name,
+      a {
+        color: var(--seeds-color-blue-900);
+      }
+    }
+
+    .listing-card-content {
+      display: flex;
+      flex-direction: row;
+      color: var(--seeds-text-color);
+      font-family: var(--seeds-font-alt-sans);
+      width: 100%;
+      @media (--md-down) {
+        flex-direction: column;
+      }
+
+      .image {
+        width: calc(100% / 7 * 3);
+        @media (--md-down) {
+          width: 100%;
+        }
+      }
+
+      .image-background {
+        height: 100%;
+        background-size: cover;
+        background-position: center;
+        @media (--md-down) {
+          min-height: var(--seeds-s96);
+        }
+        @media (--sm-only) {
+          min-height: var(--seeds-s56);
+        }
+      }
+
+      .details {
+        order: 1;
```

Comment:
nit: took me a bit to figure out how the image was getting put on the left side. It might be useful to add a comment in the `ListingCard` component to explain why the image is after the details in the code, but displayed first visually